src/app/a05c-guest-requests-landing/a05c-guest-requests-landing.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { A05cGuestRequestsLandingComponent } from './a05c-guest-requests-landing.component';
+
+describe('A05cGuestRequestsLandingComponent', () => {
+  let component: A05cGuestRequestsLandingComponent;
+  let fixture: ComponentFixture<A05cGuestRequestsLandingComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ A05cGuestRequestsLandingComponent ]
+    })
+    .compileComponents();
+
+    fixture = TestBed.createComponent(A05cGuestRequestsLandingComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
and just in case... you can "hide" a test, within its TestSuite, but adding an x to the it, so it looks like xit('should create...

then when we run the tests, it won't test..